ui/raidboss/data/04-sb/eureka/eureka_hydatos.js

```diff
@@ -13,6 +13,7 @@
       id: 'BA Art Geas',
       regex: /Legendary Geas/,
       regexDe: /Wirbelsturm/,
+      regexFr: /Tuerie spectrale/,
```

Comment:
Ah no, these are all timelineRegexes.  I'm sorry I didn't catch this on everybody else's changes before, but the correct thing is that timelineTriggers should only have regex (and no regexDe/regexFr/etc etc etc) because they are matching the timeline before it has been translated.

Reply:
I took care of this in f887686f7a016fb1f45dfeb79b3cad3c6cc36021, and added a test so this won't get added again accidentally.

Reply:
it's remove.